docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,392 @@
+- Persistent storage that will survive a VM or container rotation.
+- Machines:
+
+[%header,cols=2*]
+|===
+|Cloud Provider
+|Machine Type
+
+|Google Cloud
+|n1-standard-2
+
+|AWS
+|m5.large
+
+|Azure
+|D2s v3
+
+|Self-hosted
+|2 vCPU / 4 GiB RAM / 1 GiB Persistent Storage
+|===
+
+== Build from Source
+
+See the https://github.com/keep-network/keep-core/tree/master/docs/development#building[building] section in our developer docs.
+
+== Docker
+
+=== Get Image
+
+Doesn't exist for the public yet.
+
+=== Run Image
+This is a sample run command to for illustration purposes only.
+
+```
+export KEEP_CLIENT_ETHEREUM_PASSWORD=$(cat .secrets/eth-account-password.txt)
+export KEEP_CLIENT_CONFIG_DIR=$(pwd)/config
+export KEEP_CLIENT_PERSISTENCE_DIR=$(pwd)/persistence
+
+docker run -dit \
+--volume $KEEP_CLIENT_PERSISTENCE_DIR:/mnt/keep-client/persistence \
+--volume $KEEP_CLIENT_CONFIG_DIR:/mnt/keep-client/config \
+--env KEEP_ETHEREUM_PASSWORD=$KEEP_CLIENT_ETHEREUM_PASSWORD \
+--env LOG_LEVEL=debug \
+-p 3919:3919 \
+keep-client --config /mnt/keep-client/config/keep-client-config.toml start
+```
+
+== Configuration
+
+=== Network
+
+Default port mappings.
+
+[%header,cols=2*]
+|===
+|Egress
+|Port
+
+|Ethereum Network
+|`8545` / `8546`
+
+|Keep Network
+|`3919`
+|===
+
+[%header,cols=2*]
+|===
+|Ingress
+|Port
+
+|Keep Network
+|`3919`
+|===
+
+If you set a different `port` in your keep-client configuration, or configure `peers` with
+non-default ports configured, firewall rules will need to be adjusted accordingly.
+
+=== Application
+
+==== Sample
+
+```
+# Ethereum host connection info.
+[ethereum]
+  URL = "ws://127.0.0.1:8546"
+  URLRPC = "http://127.0.0.1:8545"
+
+# Keep operator Ethereum account.
+[ethereum.account]
+  Address = "0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8AAAAAAAAA"
+  KeyFile = "/Users/someuser/ethereum/data/keystore/UTC--2018-03-11T01-37-33.202765887Z--AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8AAAAAAAAA"
+
+# Keep contract addresses configuration.
+[ethereum.ContractAddresses]
+  # Hex-encoded address of KeepRandomBeaconOperator contract
+  KeepRandomBeaconOperator = "0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+  # Hex-encoded address of TokenStaking contract
+  TokenStaking = "0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"
+  # Hex-encoded address of KeepRandomBeaconService contract. Only needed
+  # in cases where the client's utility functions will be used (e.g., the
+  # relay subcommand).
+  KeepRandomBeaconService = "0xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"
+
+# Keep network configuration.
+[LibP2P]
+  Peers = ["/ip4/127.0.0.1/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1", "/dns4/some-keep-host.com/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1"]
+  Port  = 3920
+  # Override the node's default addresses announced in the network
+  AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
+
+[Storage]
```

Comment:
Should we emphasize the importance of backing up the storage? 

Also:
- worth explaining the structure of storage directory, what's in `current` and what's in `archive`
- worth explaining the same key is used to protect ethereum key and storage; rotating storage key is not currently supported

Reply:
Got it

Reply:
To clarify @pdyraga , `worth explaining the same key`, what key is this?

> what's in current and what's in archive

Can you give me a one liner?